packages/turbo-workspaces/src/install.ts

```diff
@@ -0,0 +1,42 @@
+import execa from "execa";
+import ora from "ora";
+import { satisfies } from "semver";
+import { ConvertError } from "./errors";
+import { PACKAGE_MANAGERS, InstallArgs } from "./types";
+
+async function install(args: InstallArgs) {
+  const { to, logger, options } = args;
+  let packageManager = PACKAGE_MANAGERS[to.name].find((manager) =>
+    satisfies(to.version, manager.semver)
```

Comment:
This kinda stuff is going to be a pain in Rust land. We're going to need `node-compatibile-semver`.

Reply:
Yea, agreed